ci/setup-build-test-gdk.ps1

```diff
@@ -27,50 +28,44 @@ Start-Event "get-unreal-engine" "command"
 &$PSScriptRoot"\get-engine.ps1" -unreal_path "$unreal_path"
 Finish-Event "get-unreal-engine" "command"
 
-Start-Event "symlink-gdk" "command"
-$gdk_in_engine = "$unreal_path\Engine\Plugins\UnrealGDK"
-New-Item -ItemType Junction -Name "UnrealGDK" -Path "$unreal_path\Engine\Plugins" -Target "$gdk_home"
-Finish-Event "symlink-gdk" "command"
-
 # Run the required setup steps
 Start-Event "setup-gdk" "command"
 &$PSScriptRoot"\setup-gdk.ps1" -gdk_path "$gdk_in_engine" -msbuild_path "$msbuild_exe"
 Finish-Event "setup-gdk" "command"
 
-# Build the GDK plugin
-&$PSScriptRoot"\build-gdk.ps1" -target_platform $($target_platform) -build_output_dir "$build_home\SpatialGDKBuild" -unreal_path $unreal_path
+# Update spatial to compatible version
+$proc = Start-Process spatial "update","$spatial_cli_version" -Wait -ErrorAction Stop -NoNewWindow -PassThru
+if ($proc.ExitCode -ne 0) {
```

Comment:
If the only reason we are using `Start-Process` is to check the exit code, you can just check `$LastExitCode`